mifospay/build.gradle

```diff
@@ -75,6 +75,8 @@ dependencies {
     implementation "io.reactivex:rxandroid:$rootProject.rxandroidVersion"
     implementation "io.reactivex:rxjava:$rootProject.rxjavaVersion"
 
+    implementation "io.michaelrocks:libphonenumber-android:8.11.0"
```

Comment:
We would have to close #567 if you add this but I think that would be wiser option. We would still cosider @Bhavnaharitsa 's contribution. So mention in the PR desc that it fixes that issue too.